docs/en/sql-reference/statements/alter/quota.md

```diff
@@ -5,16 +5,33 @@ toc_title: QUOTA
 
 # ALTER QUOTA {#alter-quota-statement}
 
-Changes quotas.
+Changes [quotas](../../../operations/access-rights.md#quotas-management).
 
 Syntax:
 
 ``` sql
 ALTER QUOTA [IF EXISTS] name [ON CLUSTER cluster_name]
     [RENAME TO new_name]
-    [KEYED BY {'none' | 'user name' | 'ip address' | 'client key' | 'client key or user name' | 'client key or ip address'}]
+    [KEYED BY {NONE | USER_NAME | IP_ADDRESS | CLIENT_KEY | CLIENT_KEY, USER_NAME | CLIENT_KEY, IP_ADDRESS} | NOT KEYED]
     [FOR [RANDOMIZED] INTERVAL number {SECOND | MINUTE | HOUR | DAY | WEEK | MONTH | QUARTER | YEAR}
-        {MAX { {QUERIES | ERRORS | RESULT ROWS | RESULT BYTES | READ ROWS | READ BYTES | EXECUTION TIME} = number } [,...] |
+        {MAX { {QUERIES | ERRORS | RESULT_ROWS | RESULT_BYTES | READ_ROWS | READ_BYTES | EXECUTION_TIME} = number } [,...] |
         NO LIMITS | TRACKING ONLY} [,...]]
     [TO {role [,...] | ALL | ALL EXCEPT role [,...]}]
 ```
+Multiword key types may be written either with underscores (`CLIENT_KEY`), or with spaces and in simple quotes (`'client key'`). You may also use `'client key or user name'` instead of `CLIENT_KEY, USER_NAME`, and `'client key or ip address'` instead of `CLIENT_KEY, IP_ADDRESS`.
```

Comment:
There were changes in the syntax, and the old syntax is still supported in the code, but I think the documentation should describe only an actual way of writing those quotas. 

Here is the actual way:

```
CREATE QUOTA q12_01297 KEYED BY client_key, ip_address
CREATE QUOTA q6_01297 FOR INTERVAL 2 month MAX queries = 100, errors = 11, result_rows = 1000, result_bytes = 10000, read_rows = 1001, read_bytes = 10001, execution_time = 2.5
```

Reply:
`KEYED BY none` is also an example of the old syntax. It should be written as `NOT KEYED` now.

Reply:
> There were changes in the syntax, and the old syntax is still supported in the code, but I think the documentation should describe only an actual way of writing those quotas.
> 
> Here is the actual way:
> 
> ```
> CREATE QUOTA q12_01297 KEYED BY client_key, ip_address
> CREATE QUOTA q6_01297 FOR INTERVAL 2 month MAX queries = 100, errors = 11, result_rows = 1000, result_bytes = 10000, read_rows = 1001, read_bytes = 10001, execution_time = 2.5
> ```

@vitlibar The idea of including both old and new syntax was to assure the users that their old queries are still okay. If it's not needed, I deleted it.